tests/aws/services/sqs/test_sqs_backdoor.py

```diff
@@ -361,3 +364,128 @@ def test_list_messages_with_queue_url_in_path(
         assert response.status_code == 400
         doc = response.json()
         assert doc["ErrorResponse"]["Error"]["Code"] == "AWS.SimpleQueueService.NonExistentQueue"
+
+
+class TestSqsOverrideHeaders:
+    @markers.aws.only_localstack
+    def test_receive_message_override_max_number_of_messages(
+        self, sqs_create_queue, aws_client_factory
+    ):
+        # Create standalone boto3 client since registering hooks to the session-wide
+        # aws_client (from the fixture) will have side-effects.
+        aws_client = aws_client_factory().sqs
+
+        override_max_number_of_messages = 20
+
+        from localstack.services.sqs.constants import HEADER_LOCALSTACK_SQS_OVERRIDE_MESSAGE_COUNT
+        from localstack.services.sqs.provider import MAX_NUMBER_OF_MESSAGES
```

Comment:
Why not import them on the top of the test function? Or on a module level altogether?